styles/_community.scss

```diff
@@ -90,11 +90,7 @@
   margin-bottom: 10px;
   width: 100%;
 
-  .dropdown {
-    width: 100%;
-  }
-
-  [role="button"] {
+  .button {
```

Comment:
Does the button have this class already, or do we need to add it in the template?

Reply:
Yup:
https://github.com/zendesk/copenhagen_theme/blob/api_v2/templates/community_topic_page.hbs#L32
https://github.com/zendesk/copenhagen_theme/blob/api_v2/templates/community_post_page.hbs#L43
https://github.com/zendesk/copenhagen_theme/blob/api_v2/templates/user_profile_page.hbs#L33